src/client/sandboxed-jquery.js

```diff
@@ -1719,7 +1719,11 @@ export function init (window, undefined) {
                     container.style.zoom = 1;
                 }
 
-                body.removeChild(container);
+                // NOTE: in some cases Firefox can't remove container div,
+                // because document body isn't appended it.
```

Comment:
> isn't appended it

what does it mean? the `div` is not in the `body`? Then it's

```
                // NOTE: in some cases, Firefox can't remove the container div,
                // because it isn't appended to the document body.
```